scripts/classes.py

```diff
@@ -0,0 +1,96 @@
+#Scopes and Namespaces Example
+
+def scope_test():
+    def do_local():
+        spam = "local spam"
+
+    def do_nonlocal():
+        nonlocal spam
+        spam = "nonlocal spam"
+
+    def do_global():
+        global spam
+        spam = "global spam"
+
+    spam = "test spam"
+    do_local()
+    print("After local assignment:", spam)#After local assignment: test spam
+    do_nonlocal()
+    print("After nonlocal assignment:", spam)#After nonlocal assignment: nonlocal spam
+    do_global()
+    print("After global assignment:", spam)#After global assignment: nonlocal spam
+
+scope_test()
+print("In global scope:", spam)#In global scope: global spam
+
+
+#Class example
+
+class MyClass():
+    def __init__(self):
+        print("This magic method invoked on object creations.")
+
+    def my_func(self):
+        print("This is class method")
+
+obj=MyClass()#Object creation
+obj.my_func()#method call
+
+#Single Inheritance
+
+class Base():
```

Comment:
As we are currently using python3 its fine, but please check what is old-style and new-style classes.

Reply:
okk